packages/frontend/src/features/sync/components/SyncModalView.tsx

```diff
@@ -18,17 +21,49 @@ import {
     IconTrash,
 } from '@tabler/icons-react';
 import cronstrue from 'cronstrue';
-import { type FC } from 'react';
+import { useState, type FC } from 'react';
 import MantineIcon from '../../../components/common/MantineIcon';
 import { useChartSchedulers } from '../../../features/scheduler/hooks/useChartSchedulers';
+import { useSchedulersEnabledUpdateMutation } from '../../scheduler/hooks/useSchedulersUpdateMutation';
 import { SyncModalAction, useSyncModal } from '../providers/SyncModalProvider';
 
+const ToggleSyncEnabled: FC<{ scheduler: Scheduler }> = ({ scheduler }) => {
+    const { mutate: mutateSchedulerEnabled } =
+        useSchedulersEnabledUpdateMutation(scheduler.schedulerUuid);
+
+    const [schedulerEnabled, setSchedulerEnabled] = useState<boolean>(
+        scheduler.enabled,
+    ); // To avoid delay on toggle
+    return (
+        <Tooltip
+            label={
+                scheduler.enabled
+                    ? 'Toggle off to temporarily pause notifications'
+                    : 'Notifications paused. Toggle on to resume'
+            }
+        >
+            <Box>
+                <Switch
+                    mr="sm"
+                    onLabel="on"
+                    offLabel="paused"
+                    checked={schedulerEnabled}
+                    onChange={() => {
+                        mutateSchedulerEnabled(!schedulerEnabled);
```

Comment:
not sure if it's been done in the other schedulers, but should we handle an error on the mutation and set the `setSchedulerEnabled` back to where it was in case it errors?